*** ./smali/com/android/server/power/DisplayPowerController.smali	2016-02-08 00:28:35.197059886 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/services.jar.out/./smali/com/android/server/power/DisplayPowerController.smali	2016-02-08 00:27:58.572944533 -0200
***************
*** 2172,2178 ****
  
      iget-object v0, p0, Lcom/android/server/power/DisplayPowerController;->mElectronBeamOffAnimator:Landroid/animation/ObjectAnimator;
  
!     const-wide/16 v1, 0x190
  
      invoke-virtual {v0, v1, v2}, Landroid/animation/ObjectAnimator;->setDuration(J)Landroid/animation/ObjectAnimator;
  
--- 2172,2178 ----
  
      iget-object v0, p0, Lcom/android/server/power/DisplayPowerController;->mElectronBeamOffAnimator:Landroid/animation/ObjectAnimator;
  
!     const-wide/16 v1, 0x0
  
      invoke-virtual {v0, v1, v2}, Landroid/animation/ObjectAnimator;->setDuration(J)Landroid/animation/ObjectAnimator;
  
***************
*** 2594,2601 ****
  
      invoke-virtual {v0}, Lcom/android/server/power/Notifier;->onScreenOn()V
  
-     :cond_0
      :goto_0
      return-void
  
      :cond_1
--- 2594,2603 ----
  
      invoke-virtual {v0}, Lcom/android/server/power/Notifier;->onScreenOn()V
  
      :goto_0
+     invoke-static {p1}, Lcom/android/server/power/ButtonLightController;->setScreenOn(Z)V
+ 
+     :cond_0
      return-void
  
      :cond_1
