*** ./smali/android/telephony/SmsMessage.smali	2016-02-08 00:28:32.629051795 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/telephony-common.jar.out/./smali/android/telephony/SmsMessage.smali	2016-02-08 00:27:55.924936197 -0200
***************
*** 829,834 ****
--- 829,847 ----
      return-object v0
  .end method
  
+ .method public getEncodingType()I
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/telephony/SmsMessage;->mWrappedSmsMessage:Lcom/android/internal/telephony/SmsMessageBase;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/SmsMessageBase;->getEncodingType()I
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public getIndexOnIcc()I
      .locals 1
  
