*** ./smali/android/preference/RingtonePreference.smali	2016-02-08 00:28:20.213012680 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2016-02-08 00:27:43.384896724 -0200
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.activity.ThemeTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
