*** ./smali/android/media/MediaMetadataEditor.smali	2016-02-08 00:28:29.777042810 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/media/MediaMetadataEditor.smali	2016-02-08 00:27:53.052927155 -0200
***************
*** 140,145 ****
--- 140,151 ----
  
      sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
  
+     const/16 v1, 0x3e8
+ 
+     invoke-virtual {v0, v1, v2}, Landroid/util/SparseIntArray;->put(II)V
+ 
+     sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
+ 
      const/16 v1, 0x64
  
      invoke-virtual {v0, v1, v5}, Landroid/util/SparseIntArray;->put(II)V
