*** ./smali/android/view/LayoutInflater.smali	2016-02-08 00:28:18.121006091 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/view/LayoutInflater.smali	2016-02-08 00:27:40.812888630 -0200
***************
*** 1533,1538 ****
--- 1533,1546 ----
  
      move-result-object v1
  
+     invoke-static {v1, p1}, Landroid/view/LayoutInflaterMap;->getResourceId(Landroid/content/Context;I)I
+ 
+     move-result p1
+ 
+     invoke-virtual {p0}, Landroid/view/LayoutInflater;->getContext()Landroid/content/Context;
+ 
+     move-result-object v1
+ 
      invoke-virtual {v1}, Landroid/content/Context;->getResources()Landroid/content/res/Resources;
  
      move-result-object v1
