*** ./smali/android/view/ViewDebug.smali	2016-02-08 00:28:17.613004491 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/view/ViewDebug.smali	2016-02-08 00:27:40.336887132 -0200
***************
*** 4688,4693 ****
--- 4688,4701 ----
      const-string v3, "\\n"
  
      invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
+     const-string v2, "\r"
+ 
+     const-string v3, "\\r"
+ 
+     invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
      :try_end_0
      .catchall {:try_start_0 .. :try_end_0} :catchall_0
  
