*** ./smali/android/app/ApplicationPackageManager.smali	2016-02-08 00:28:13.988993076 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/app/ApplicationPackageManager.smali	2016-02-08 00:27:37.028876721 -0200
***************
*** 1466,1471 ****
--- 1466,1480 ----
      goto :goto_0
  .end method
  
+ .method getContext()Landroid/app/ContextImpl;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/app/ApplicationPackageManager;->mContext:Landroid/app/ContextImpl;
+ 
+     return-object v0
+ .end method
+ 
  .method public getDefaultActivityIcon()Landroid/graphics/drawable/Drawable;
      .locals 2
  
***************
*** 2670,2675 ****
--- 2679,2688 ----
      move-result-object v0
  
      .local v0, "r":Landroid/content/res/Resources;
+     iget-object v1, p1, Landroid/content/pm/ApplicationInfo;->packageName:Ljava/lang/String;
+ 
+     invoke-static {v0, v1}, Landroid/miui/ResourcesManager;->initMiuiResource(Landroid/content/res/Resources;Ljava/lang/String;)V
+ 
      if-nez v0, :cond_0
  
      new-instance v1, Landroid/content/pm/PackageManager$NameNotFoundException;
