*** ./smali/android/app/LoadedApk.smali	2016-02-08 00:28:13.352991074 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/app/LoadedApk.smali	2016-02-08 00:27:36.396874733 -0200
***************
*** 1812,1817 ****
--- 1812,1823 ----
  
      iput-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
+     iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
+ 
+     iget-object v1, p0, Landroid/app/LoadedApk;->mPackageName:Ljava/lang/String;
+ 
+     invoke-static {v0, v1}, Landroid/miui/ResourcesManager;->initMiuiResource(Landroid/content/res/Resources;Ljava/lang/String;)V
+ 
      :cond_0
      iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
