*** ./smali/com/android/internal/statusbar/IStatusBar.smali	2016-02-08 00:28:39.165072390 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework2.jar.out/./smali/com/android/internal/statusbar/IStatusBar.smali	2016-02-08 00:28:03.540960175 -0200
***************
*** 111,116 ****
--- 111,124 ----
      .end annotation
  .end method
  
+ .method public abstract setStatus(ILjava/lang/String;Landroid/os/Bundle;)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setSystemUiVisibility(II)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
