*** ./smali/android/webkit/WebView.smali	2016-02-08 00:28:06.160968425 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/webkit/WebView.smali	2016-02-08 00:27:30.960857628 -0200
***************
*** 167,172 ****
--- 167,176 ----
  
      invoke-static {}, Landroid/webkit/CookieSyncManager;->setGetInstanceIsAllowed()V
  
+     invoke-static {p0, p1}, Landroid/webkit/WebViewInjector;->initEgretLoader(Landroid/webkit/WebView;Landroid/content/Context;)V
+ 
+     invoke-static {p1}, Lmiui/content/res/ThemeFontChangeHelper;->markWebViewCreated(Landroid/content/Context;)V
+ 
      return-void
  
      :cond_1
