*** ./smali/android/net/wifi/ScanResult$1.smali	2016-02-08 00:28:26.557032664 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/net/wifi/ScanResult$1.smali	2016-02-08 00:27:49.912917271 -0200
***************
*** 40,46 ****
  
  # virtual methods
  .method public createFromParcel(Landroid/os/Parcel;)Landroid/net/wifi/ScanResult;
!     .locals 10
      .param p1, "in"    # Landroid/os/Parcel;
  
      .prologue
--- 40,46 ----
  
  # virtual methods
  .method public createFromParcel(Landroid/os/Parcel;)Landroid/net/wifi/ScanResult;
!     .locals 12
      .param p1, "in"    # Landroid/os/Parcel;
  
      .prologue
***************
*** 96,102 ****
  
      move-result v9
  
!     invoke-direct/range {v0 .. v9}, Landroid/net/wifi/ScanResult;-><init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJII)V
  
      return-object v0
  .end method
--- 96,110 ----
  
      move-result v9
  
!     invoke-virtual {p1}, Landroid/os/Parcel;->readString()Ljava/lang/String;
! 
!     move-result-object v10
! 
!     invoke-virtual {p1}, Landroid/os/Parcel;->readString()Ljava/lang/String;
! 
!     move-result-object v11
! 
!     invoke-direct/range {v0 .. v11}, Landroid/net/wifi/ScanResult;-><init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJIILjava/lang/String;Ljava/lang/String;)V
  
      return-object v0
  .end method
