*** ./smali/android/net/wifi/ScanResult.smali	2016-02-08 00:28:27.429035412 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/net/wifi/ScanResult.smali	2016-02-08 00:27:50.748919903 -0200
***************
*** 40,45 ****
--- 40,49 ----
  
  .field public wifiSsid:Landroid/net/wifi/WifiSsid;
  
+ .field public wpsDeviceName:Ljava/lang/String;
+ 
+ .field public wpsState:Ljava/lang/String;
+ 
  
  # direct methods
  .method static constructor <clinit>()V
***************
*** 100,105 ****
--- 104,117 ----
  
      iput v0, p0, Landroid/net/wifi/ScanResult;->distanceSdCm:I
  
+     iget-object v0, p1, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iput-object v0, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iget-object v0, p1, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
+     iput-object v0, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
      :cond_0
      return-void
  .end method
***************
*** 143,148 ****
--- 155,166 ----
  
      iput v1, p0, Landroid/net/wifi/ScanResult;->distanceSdCm:I
  
+     const/4 v1, 0x0
+ 
+     iput-object v1, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iput-object v1, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
      return-void
  
      :cond_0
***************
*** 151,156 ****
--- 169,195 ----
      goto :goto_0
  .end method
  
+ .method public constructor <init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJLjava/lang/String;Ljava/lang/String;)V
+     .locals 2
+     .param p1, "wifiSsid"    # Landroid/net/wifi/WifiSsid;
+     .param p2, "BSSID"    # Ljava/lang/String;
+     .param p3, "caps"    # Ljava/lang/String;
+     .param p4, "level"    # I
+     .param p5, "frequency"    # I
+     .param p6, "tsf"    # J
+     .param p8, "wpsState"    # Ljava/lang/String;
+     .param p9, "wpsDeviceName"    # Ljava/lang/String;
+ 
+     .prologue
+     invoke-direct/range {p0 .. p7}, Landroid/net/wifi/ScanResult;-><init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJ)V
+ 
+     iput-object p8, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iput-object p9, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
+     return-void
+ .end method
+ 
  .method public constructor <init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJII)V
      .locals 1
      .param p1, "wifiSsid"    # Landroid/net/wifi/WifiSsid;
***************
*** 190,195 ****
--- 229,240 ----
  
      iput p9, p0, Landroid/net/wifi/ScanResult;->distanceSdCm:I
  
+     const/4 v0, 0x0
+ 
+     iput-object v0, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iput-object v0, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
      return-void
  
      :cond_0
***************
*** 198,203 ****
--- 243,271 ----
      goto :goto_0
  .end method
  
+ .method public constructor <init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJIILjava/lang/String;Ljava/lang/String;)V
+     .locals 1
+     .param p1, "wifiSsid"    # Landroid/net/wifi/WifiSsid;
+     .param p2, "BSSID"    # Ljava/lang/String;
+     .param p3, "caps"    # Ljava/lang/String;
+     .param p4, "level"    # I
+     .param p5, "frequency"    # I
+     .param p6, "tsf"    # J
+     .param p8, "distCm"    # I
+     .param p9, "distSdCm"    # I
+     .param p10, "wpsState"    # Ljava/lang/String;
+     .param p11, "wpsDeviceName"    # Ljava/lang/String;
+ 
+     .prologue
+     invoke-direct/range {p0 .. p9}, Landroid/net/wifi/ScanResult;-><init>(Landroid/net/wifi/WifiSsid;Ljava/lang/String;Ljava/lang/String;IIJII)V
+ 
+     iput-object p10, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     iput-object p11, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
+     return-void
+ .end method
+ 
  
  # virtual methods
  .method public describeContents()I
***************
*** 357,362 ****
--- 425,452 ----
  
      invoke-virtual {v2, v3}, Ljava/lang/StringBuffer;->append(Ljava/lang/String;)Ljava/lang/StringBuffer;
  
+     const-string v2, ", wpsState :"
+ 
+     invoke-virtual {v1, v2}, Ljava/lang/StringBuffer;->append(Ljava/lang/String;)Ljava/lang/StringBuffer;
+ 
+     move-result-object v2
+ 
+     iget-object v3, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     invoke-virtual {v2, v3}, Ljava/lang/StringBuffer;->append(Ljava/lang/String;)Ljava/lang/StringBuffer;
+ 
+     move-result-object v2
+ 
+     const-string v3, ", wpsDeviceName :"
+ 
+     invoke-virtual {v2, v3}, Ljava/lang/StringBuffer;->append(Ljava/lang/String;)Ljava/lang/StringBuffer;
+ 
+     move-result-object v2
+ 
+     iget-object v3, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
+     invoke-virtual {v2, v3}, Ljava/lang/StringBuffer;->append(Ljava/lang/String;)Ljava/lang/StringBuffer;
+ 
      invoke-virtual {v1}, Ljava/lang/StringBuffer;->toString()Ljava/lang/String;
  
      move-result-object v2
***************
*** 438,443 ****
--- 528,541 ----
  
      invoke-virtual {p1, v0}, Landroid/os/Parcel;->writeInt(I)V
  
+     iget-object v0, p0, Landroid/net/wifi/ScanResult;->wpsState:Ljava/lang/String;
+ 
+     invoke-virtual {p1, v0}, Landroid/os/Parcel;->writeString(Ljava/lang/String;)V
+ 
+     iget-object v0, p0, Landroid/net/wifi/ScanResult;->wpsDeviceName:Ljava/lang/String;
+ 
+     invoke-virtual {p1, v0}, Landroid/os/Parcel;->writeString(Ljava/lang/String;)V
+ 
      return-void
  
      :cond_0
