*** ./smali/android/net/wifi/SupplicantStateTracker$DefaultState.smali	2016-02-08 00:28:26.645032942 -0200
--- /home/kowak/patchrom/condor/temp/new_smali/framework.jar.out/./smali/android/net/wifi/SupplicantStateTracker$DefaultState.smali	2016-02-08 00:27:49.996917536 -0200
***************
*** 151,156 ****
--- 151,160 ----
      # setter for: Landroid/net/wifi/SupplicantStateTracker;->mAssociationRejectCount:I
      invoke-static {v2, v4}, Landroid/net/wifi/SupplicantStateTracker;->access$702(Landroid/net/wifi/SupplicantStateTracker;I)I
  
+     iget v2, p1, Landroid/os/Message;->arg1:I
+ 
+     invoke-static {v2}, Landroid/net/wifi/SupplicantStateTrackerInjector;->handleConnectNetwork(I)V
+ 
      goto :goto_0
  
      :sswitch_4
